utils/build_swift/driver_arguments.py

```diff
@@ -0,0 +1,1295 @@
+
+    build_variant_choices = [
+        'Debug',
+        'RelWithDebInfo',
+        'Release',
+        'MinSizeRel',
+    ]
+
+    with mutually_exclusive_group():
+        option('--debug-cmark',
+               set_('cmark_build_variant', const='Debug'),
+               help='build the Debug variant of CommonMark')
+        option('--cmark-build-variant', set_,
+               choices=build_variant_choices,
+               help='CMake build variant for CommonMark')
+
+    with mutually_exclusive_group():
+        option('--debug-foundation',
+               set_('foundation_build_variant', const='Debug'),
+               help='build the Debug variant of Foundation')
+        option('--foundation-build-variant', set_,
+               choices=build_variant_choices,
+               help='CMake build variant for Foundation')
+
+    with mutually_exclusive_group():
+        option('--debug-llbuild',
+               set_('llbuild_build_variant', const='Debug'),
+               help='build the Debug variant of llbuild')
+        option('--llbuild-build-variant', set_,
+               choices=build_variant_choices,
+               help='CMake build variant for llbuild')
+
+    with mutually_exclusive_group():
+        option('--debug-lldb',
+               set_('lldb_build_variant', const='Debug'),
+               help='build the Debug variant of LLDB')
+        option('--lldb-build-variant', set_,
+               choices=build_variant_choices,
+               help='CMake build variant for LLDB')
+
+    with mutually_exclusive_group():
+        option('--debug-libdispatch',
+               set_('libdispatch_build_variant', const='Debug'),
+               help='build the Debug variant of libdispatch')
+        option('--libdispatch-build-variant', set_,
+               choices=build_variant_choices,
+               help='CMake build variant for libdispatch')
+
+    with mutually_exclusive_group():
+        option('--debug-libicu',
+               set_('libicu_build_variant', const='Debug'),
+               help='build the Debug variant of libdicu')
+        option('--libdicu-build-variant', set_,
+               choices=build_variant_choices,
+               help='CMake build variant for libicu')
+
+    with mutually_exclusive_group():
+        option('--debug-llvm',
+               set_('llvm_build_variant', const='Debug'),
+               help='build the Debug variant of LLVM')
+        option('--llvm-build-variant', set_,
+               choices=build_variant_choices,
+               help='CMake build variant for LLVM and Clang')
+
+    with mutually_exclusive_group():
+        option('--debug-swift',
+               set_('swift_build_variant', const='Debug'),
+               help='build the Debug variant of Swift')
+        option('--swift-build-variant', set_,
+               choices=build_variant_choices,
+               help='CMake build variant for Swift')
+
+    with mutually_exclusive_group():
+        option('--debug-swift-stdlib',
```

Comment:
Can you make it clearer here that "the swift compiler" is not controlled by this option?